README.md

```diff
@@ -709,6 +710,20 @@ $ remme transaction get-list \
 }
 ```
 
+Get only a list of transaction's identifiers:
```

Comment:
`Get only a list of transaction's identifiers` > `Get a list of transaction's identifiers`.